fvtest/lljbtest/lljb_run.cpp

```diff
@@ -0,0 +1,61 @@
+/*******************************************************************************
+ * Copyright (c) 2020, 2020 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+
+#include "lljb/Module.hpp"
+#include "lljb/Compiler.hpp"
+#include "lljb/IRVisitor.hpp"
+#include "JitBuilder.hpp"
+
+#include "llvm/IR/Module.h"
+#include "llvm/IR/LLVMContext.h"
+#include "llvm/Support/SourceMgr.h"
+
+#include <iostream>
+
+int main(int argc, char * argv[])
+   {
+   if (argc != 2){
+      std::cerr << "Usage: ./lljb <module.ll>" << std::endl;
+      exit(EXIT_FAILURE);
+   }
+
+   if (!initializeJit()){
+      std::cerr << "Failed to initialize JIT" << std::endl;
+      exit(EXIT_FAILURE);
+   }
```

Comment:
Whitespace nitpick, hope I'm not annoying yet :)

```suggestion
   if (argc != 2)
      {
      std::cerr << "Usage: ./lljb <module.ll>" << std::endl;
      exit(EXIT_FAILURE);
      }

   if (!initializeJit())
      {
      std::cerr << "Failed to initialize JIT" << std::endl;
      exit(EXIT_FAILURE);
      }
```

Reply:
Good catch once again! :)